tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -638,42 +638,40 @@ public function test_old_and_new_in_footer_scripts() {
 	 */
 	public function test_get_normalized_script_args() {
 		global $wp_scripts;
+
+		// Passing in_footer and strategy via args array.
 		$args = array(
 			'in_footer' => true,
 			'strategy'  => 'async',
 		);
 		wp_enqueue_script( 'footer-async', '/footer-async.js', array(), null, $args );
-		$this->assertSame( $args, $wp_scripts->get_data( 'footer-async', 'script_args' ) );
+		$this->assertSame( $args['in_footer'], (bool) $wp_scripts->get_data( 'footer-async', 'group' ) );
```

Comment:
```suggestion
		$this->assertSame( 1, $wp_scripts->get_data( 'footer-async', 'group' ) );
```
@westonruter  in this case group would be 1 and I feel we should `assetSame` as `1` instead of against `$args['in_footer']`. Even though they are correlated and contains similar values, `in_footer` and `group` are not the same. A group value of `2` will still result in asserting true in the prior scenario.

Reply:
Good catch.

Reply:
Thanks @kt-12